websites/C/Cytoid/dist/metadata.json

```
@@ -0,0 +1,43 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "joerkig",
+		"id": "205984221859151873"
+	},
+	"service": "Cytoid",
+	"description": {
+		"en": "Cytoid is a music game where you can create, share and play your own levels! Powered by community, with many dedicated creators, Cytoid provides a huge variety of musical genres to enjoy and a diverse range of gameplay design."
+	},
+	"url": "cytoid.io",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/3K3skPd.png",
+	"thumbnail": "https://i.imgur.com/oOrYTUr.png",
+	"color": "#476ADC",
+	"category": "games",
+	"tags": [
+		"rhythm",
+		"music",
+		"gaming",
+		"touch"
+	],
+	"settings": [
+		{
+			"id": "time",
```

I think this would be more consistent with the repo
```suggestion
			"id": "timestamps",
```